README.md

```diff
@@ -296,11 +229,37 @@ account.
     ```
 
 Alternatively, create an `environment.local.py` file in the project root
-directory and specifiy a global default for `GOOGLE_APPLICATION_CREDENTIALS`
+directory and specify a global default for `GOOGLE_APPLICATION_CREDENTIALS`
 there.
 
+### 2.3.3 Google OAuth credentials
```

Comment:
Now you moved this back to the wrong place. You have to realize that developers may not have a deployment yet when they're at this stage of the README.